models/order_list.rb

```diff
@@ -0,0 +1,16 @@
+require "sequel"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.